web/client/components/background/BackgroundDialog.jsx

```diff
@@ -323,6 +323,13 @@ export default class BackgroundDialog extends React.Component {
                     map={{
                         newThumbnail: get(this.state.thumbnail, 'url') || "NODATA"
                     }}
+                    thumbnailOptions={{
+                        width: 98,
+                        height: 98,
+                        type: 'image/jpeg',
+                        quality: 0.5,
+                        contain: false
```

Comment:
Maybe we could use 128px x 128px with 0.8 quality and change the UI to be squared, something like this:

![image](https://github.com/geosolutions-it/MapStore2/assets/19175505/261219f5-c0d0-42de-a0ef-ec61c1f32e4b)

We could also change the message into: `Size of images should be a square of 128px x 128px, max 500kb`

Just tested a conversion of a [colorful image](https://unsplash.com/photos/multicolored-painting-pCcGpVsOHoo) (original size ~ 3.4 MB) into a thumbnail 128px x 128px quality 0.8. The result is an image of ~ 8.4 KB

![test](https://github.com/geosolutions-it/MapStore2/assets/19175505/7cc089c4-212f-4a27-9f28-2777d6ef90f4)